release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -341,6 +341,26 @@ The Cloud Credential Operator utility (`ccoctl`) now creates secrets that use re
 
 With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
 
+[discrete]
+[id="ocp-4-12-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
+
+To prepare for this upcoming change, ensure that your workloads match the pod security admission profile that applies to them. Workloads that are not configured according to the enforced security standards defined globally or at the namespace level will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
+
+If you are receiving pod security violations, see the following resources:
+
+* See xref:../authentication/understanding-and-managing-pod-security-admission.adoc#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission[Identifying pod security violations] for information about how to find which workloads are causing pod security violations.
```

Comment:
Interestingly, there is an "Identifying pod security violations" procedure module called `modules/security-context-constraints-psa-alert-eval.adoc` but I don't see it rendered in your build preview. But it _is_ in the pull request where you added it in 3 months ago in https://github.com/openshift/openshift-docs/pull/50631. 

Am I missing something?

Reply:
That's really weird, I swear I rebased and that link worked. I will try rebasing again to make sure it links properly before merging. Thanks!

Reply:
Rebased and it's good now. Thanks again!